ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### General information
+*(Please answer all the questions below and delete this line before submitting the issue.)*
```

Comment:
instead of asking to delete this line, wrap it in html comment, like `<!-- Please answer ..... -->`

Reply:
Woah, I didn't know that was possible 😮